### PR TITLE
Updated RegEx

### LINKED ISSUE #79: Bug in SignUp Form
#### Current Behavior
As mentioned by Lee on Slack, there's a bug in the Name field of signup form that requires users to enter the name in a specific format.
image

#### Expected Behavior
We can either remove the check since name need not follow a specific format OR make it more inclusive to include all languages

#### Environment

- **Host OS:** Windows

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by BabyElias**
**Notes for Reviewers**

This PR fixes #79 

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Avi-88** [comment]
Hi @BabyElias , the regex you proposed allows numbers as well as white space though. Also the same regex needs to be applied to last name input as well

**Comment by BabyElias** [comment]
> Hi @BabyElias , the regex you proposed allows numbers as well as white space though. Also the same regex needs to be applied to last name input as well

I tested it for numbers and whitespace, it doesn't accept them as answers.

**Comment by Avi-88** [comment]
Ohh, hmm thats wierd. Well if there isn't any problem then awesome.

**Comment by BabyElias** [comment]
@Avi-88 The name field doesn't allow for apostrophes, is that a problem?

**Comment by Avi-88** [comment]
@BabyElias , I think there's some issue
image

File: site/src/components/SignupForm/index.js [hunk 1]
```diff
@@ -111,7 +111,7 @@ const SignupForm = () => {
                   id="firstname"
                   name="firstname"
                   maxLength="32"
-                  pattern="[A-Za-z]{1,32}"
+                  pattern="([A-Za-zŽžÀ-ÿ]+('[A-Za-zÀ-ÿ])?[A-Za-zŽžÀ-ÿ])|[[A-Za-zŽžÀ-ÿ]"
```
Comment (Avi-88) [inline site/src/components/SignupForm/index.js:114]: I think there is some issue with the current one, I've added a ss in the comments. I'll recommend one more `/^[a-zA-Z'’\- ]+$/` here. Also after you finalize a regex could you please use these names to test it out , they don't cover everything but this should suffice, thanks

José Garcia-Hernández
Priya Patel-Joshi
Mohammed Al-Fayed
Yvonne D'Souza
Wei Zhang-Li
Evelyn O'Malley-Smith"

**Review by Avi-88 — CHANGES_REQUESTED** [review]

**Comment by BabyElias** [comment]
Yup, I'll check and let you know once the changes are done.

File: site/src/components/SignupForm/index.js (hunk 1 above)
Comment (BabyElias) [reply site/src/components/SignupForm/index.js]: Updated to a new regex that matches all the names you have given. This regex now accounts for presence of an apostrophe as well as a hyphen in between the name

**Comment by BabyElias** [comment]
@Avi-88 Done with changes pertaining to making the regex more inclusive

**Review by Avi-88 — APPROVED** [review]
Good work @BabyElias !

**Pull request merged**